lib/ws-kernel.js

```diff
@@ -122,7 +122,7 @@ export default class WSKernel extends Kernel {
         defaultText: this.session.path,
         allowCancel: true
       },
-      (input: string) => this.session.rename(input)
+      (input: string) => this.session.setPath(input)
```

Comment:
This doesn't change behavior. Though we should now use the session name in the UI instead of the notebook path.

Reply:
We use the name unless it is an empty string and then defer to the `basename` of the path.